### PR TITLE
Remove builtins directive

### PR DESCRIPTION
**Pull request by BitBaseBit**

### CODE REVIEW

File: tests/golden/c2c_marked.cairo [hunk 1]
```diff
@@ -0,0 +1,476 @@
+%lang starknet
```
Comment (murcake) [inline tests/golden/c2c_marked.cairo:1]: This file shouldn't be there

File: tests/behaviour/malicious-calldata_test.py [hunk 2]
```diff
@@ -1,109 +0,0 @@
-import tempfile
```
Comment (murcake) [inline tests/behaviour/malicious-calldata_test.py:1]: Why is this file deleted?..

File: pyproject.toml [hunk 3]
```diff
@@ -24,7 +24,7 @@ skip_gitignore = true
 
 [tool.poetry]
 name = "sol-warp"
-version = "0.2.12"
```
[the remark this answers is not in the document]
Comment (BitBaseBit) [reply pyproject.toml:27]: Because we've been forgetting to bump it for months lol

[the remark this answers is not in the document]
Comment (murcake) [reply pyproject.toml:27]: We didn't forget, it's just that there haven't been a release. It's normal to do a lot of changes without bumping the version until the new release

File: tests/behaviour/malicious-calldata_test.py (hunk 2 above)
Comment (BitBaseBit) [reply tests/behaviour/malicious-calldata_test.py:1]: Because it was written in a way that was completely incongruent with the style of all the other tests.

Comment (murcake) [reply tests/behaviour/malicious-calldata_test.py:1]: But it needs to be there, it tests important things?.. The whole Uint256 library is based on the premise that it's given valid Uint256's, so we need to check for that. It's written that way, because unlike other tests it tests failure.

Comment (murcake) [reply tests/behaviour/malicious-calldata_test.py:1]: It also uses fixtures, in order to avoid computing the same things again and again

Comment (BitBaseBit) [reply tests/behaviour/malicious-calldata_test.py:1]: Fair enough, I'll add it back :).

**Review by murcake — APPROVED** [review]

**Comment by murcake** [comment]
Let's rebase onto develop first, though
